.github/actions/deploy-happy-stack/action.yml

```diff
@@ -57,7 +57,16 @@ runs:
         HAPPY_DEPLOYMENT_FLAGS=""
 
         if [[ ${OPERATION} == "create-or-update" ]]; then
-          if $(happy list --aws-profile "" --env=${ENV} &>out.txt; grep "^|\s*${STACK_NAME}\s*|" out.txt); then
+          happy list --aws-profile "" --env=${ENV} &>out.txt
+          EXIT_CODE=$?
+          if [[ $EXIT_CODE -ne 0 ]]; then
+              cat out.txt
+              exit $EXIT_CODE
+          fi
+
+          STACKS=$(grep "^|\s*${STACK_NAME}\s*|" out.txt)
+         
+          if [[ $STACKS ]]; then
```

Comment:
should this be an `if [[ -n $STACKS ]]; then` ? My bash is terrible though

Reply:
I've changed it to `[[ -n "${STACKS}" ]]` as you requested, but `[[ ! -z "${STACKS}" ]]` and `[[ $STACKS ]];` behave identically with the rest of the script -- verified locally with bash and sh.